config/locales/en.yml

```diff
@@ -1594,6 +1594,90 @@ en:
       greeting: "Hi,"
       hopefully_you: "Someone (hopefully you) would like to change their email address over at %{server_url} to %{new_address}."
       click_the_link: "If this is you, please click the link below to confirm the change."
+    welcome_email:
+      subject: "[OpenStreetMap] Welcome!"
```

Comment:
I am not really sure how to reduce the duplication of messages for welcome email welcome screen.

It looks like the template files need to be stored in specific directories: `views/user_mailer/welcome_email.*.erb` and `views/site/welcome.html.erb` - so we cannot reuse the same template file.

I could potentially reuse the strings - use `layouts/site/welcome` strings instead of `layouts/user_mailer/welcome_email` in `views/user_mailer/welcome_email.*.erb`, but that didn't seem to be clean.